### PR TITLE
Ref #5023: Add copy for ticket #5023.

### PR DESCRIPTION
**Pull request by iccub**
This is so we can order translations before code for #5023 is ready.

## Summary of Changes

This pull request references #5023 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

**Review by soner-yuksel — APPROVED** [review]

File: BraveWallet/WalletStrings.swift [hunk 1]
```diff
@@ -1779,5 +1779,33 @@ extension Strings {
...
+      bundle: .braveWallet,
+      value: "You're attempting to start a new transaction while you have previously submitted transactions that have not been confirmed. This will block any new ones from being submitted.",
+      comment: "Text body of confirmation prompt when there's a backlog of wallet transactions"
+    )
+    public static let transactionBacklogAcknowledgement = NSLocalizedString(
+      "wallet.transactionBacklogAcknowledgement",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Clear & replace the incomplete transaction(s)",
+      comment: "Text of checkbox for a confirmation prompt when there's a backlog of wallet transactions"
```
Comment (nuo-xu) [inline BraveWallet/WalletStrings.swift]: iOS will have a toggle instead of a checkbox

File: BraveWallet/WalletStrings.swift [hunk 2]
```diff
@@ -1779,5 +1779,33 @@ extension Strings {
...
+    )
+    public static let transactionBacklogLearnMoreButton = NSLocalizedString(
+      "wallet.transactionBacklogLearnMoreButton",
+      tableName: "BraveWallet",
+      bundle: .braveWallet,
+      value: "Learn More.",
```
Comment (nuo-xu) [inline BraveWallet/WalletStrings.swift]: Don't think we want a dot in the end. But if copy writer approved it. I'm fine.

**Review by nuo-xu — CHANGES_REQUESTED** [review]

File: BraveWallet/WalletStrings.swift (hunk 1 above)
Comment (iccub) [reply BraveWallet/WalletStrings.swift]: Will change, this is a note for translators only though to know the context

File: BraveWallet/WalletStrings.swift (hunk 2 above)
Comment (iccub) [reply BraveWallet/WalletStrings.swift]: per slack convo we are leaving the dot. 
If needed there is a already localized learn-more string without the dot

**Review by nuo-xu — APPROVED** [review]

**Pull request merged**